Drop `dev` dependencies from pyproject.toml (#1843)
PlasmaPy/PlasmaPy: docs/contributing/install_dev.rst

Now that you have plasmapy on your local computer and you have a
virtual environment, you will want to "install" this development
version of PlasmaPy along with its dependencies. Start by activating
your virtual environment.

> Then, setup the development version of PlasmaPy which you just cloned

Reviewer: ```suggestion
Then, set up the development version of PlasmaPy which you just cloned
```

by moving into the root directory of the cloned repo and running the
following there:

.. code-block:: bash

   (plasmapy) user@name:~/programming/plasmapy$ pip install --editable ".[tests,docs]"

You should now be all set to run development versions of PlasmaPy

> modules via ``import PlasmaPy`` in your test scripts!

Reviewer: ```suggestion
modules via ``import plasmapy`` in your test scripts!
```

The ``--editable`` flag will create a soft link to your cloned
repository.  Any changes in Python code you make there will be there
when you ``import plasmapy`` from an interactive session.

>    If you are not working within a virtual environment, this may end in
>    a permission error - this can be avoided via also adding the
>    ``--user`` flag. But seriously, use a virtual environment and spare
>    yourself the trouble.

Reviewer: ```suggestion
   If you are not working within a virtual environment, this may end in
   a permission error - this can be avoided via also adding the
   ``--user`` flag. But seriously, use a virtual environment and spare
   yourself the trouble.
```
```suggestion
.. tip::

   If you are not working within a virtual environment, this may end in
   a permission error — this can be avoided via also adding the
   ``--user`` flag. But seriously, use a virtual environment and spare
   yourself the trouble.
```
